applications/web/form.yaml

```diff
@@ -232,6 +232,28 @@ tabs:
       placeholder: "ex: 30"
       settings:
         default: 30
+  - name: lifecycle_post_start_command
```

Comment:
Can we move these into the same section with the heading "Container Hooks"? Instead of two different sections with headings.

Reply:
Agreed 👌